common/src/main/resources/db/migration/mysql/V2__Setup_mixit16.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO mixit.Event (ID,CURRENT,YEAR) VALUES (6, TRUE ,2016);
```

Comment:
For the moment we have no data for 2016. So if we mark 2016 as current, the session list, speakers will be empty. We can hide this menu items for the moment.